analytics/stats.py

```diff
@@ -0,0 +1,40 @@
+import json
+import pandas as pd
+from departements import to_departement_number, import_departements
+import matplotlib.pyplot as plt
+import numpy as np
+
+def my_fmt(x):
+    print(x)
+    return '{:.4f}%\n({:.0f})'.format(x, total*x/100)
```

Comment:
Je pense que tu peux remplacer par : 
```suggestion
    return (
        f'{x:.2f}%\n'
        f'({total * x /100:.0f})'
    )
```

J'ai mis sur deux lignes pour plus de visibilité, et j'ai enlevé des chiffres significatifs qui n'ont pas trop d'intérêts à mon avis